src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo">
+        <a href="/#home">
+          <img
+            src="./images/logo.png"
+            alt="Company logo"
+        />
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Use `#` for anchor links. Also, make sure the `alt` attribute describes the image content more specifically.